dev-python/matplotlib2tikz/files/matplotlib2tikz-0.6.7-init_pipdated.patch

```diff
@@ -0,0 +1,13 @@
+Remove version checks using pipdated.
```

Comment:
One more request: please update the patch to remove `pipdated` dependency from setup.py as well. Do that as a separate PR, and no need to hurry -- you can just note that in your TODO for next bump ;-). It doesn't strictly break anything at the moment but the dependency is recorded in the installed package, so setuptools can go weird about it.